### DISCUSSION TITLE
How To Disable Google As Default GOOGLE Handler?

### DISCUSSION THREAD

**Discussion by sapphired18** [MustardChef/WSABuilds#148] [Q&A]
How to disable this? I am using discord and when i search something using context menu in discord using google, it opens up the WSA Google and then open the web browser?

Instead of using google as a proxy, I want it directly to my default browser
![image](https://github.com/MustardChef/WSABuilds/assets/58063311/2fdd83bf-cf52-4163-abb0-9f2584ad925d)

![image](https://github.com/MustardChef/WSABuilds/assets/58063311/4d42db35-514d-4268-9f95-99dad9740230)

**Comment by MustardChef** [comment]
@pc-v2 
Go to Android settings (wsa://com.android.settings), "Apps", find the installed app (in this case Google), and go to "Open by default" and then make sure "Open supported link" option is Disabled (which should be enabled by default).

**Reply by nhan000** [reply]
Post the Discord [conversation](https://discord.com/channels/1063231057278808165/1096705940285304902/1147789841187209326) as a data point that this doesn't work :-/

**Comment by nhan000** [comment]
Hi @pc-v2, were you able to find a fix?
